source/client-side-encryption/tests/unified/createKey-provider-invalid.yml

```diff
@@ -0,0 +1,71 @@
+description: createKey-provider-invalid
+
+schemaVersion: "1.8"
+
+runOnRequirements:
+  - csfle: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
+  - clientEncryption:
+      id: &clientEncryption0 clientEncryption0
+      clientEncryptionOpts:
+        keyVaultClient: *client0
+        keyVaultNamespace: keyvault.datakeys
+        kmsProviders:
+          aws:
+            accessKeyId: { $$placeholder: 1 }
+            secretAccessKey: { $$placeholder: 1 }
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &keyvault keyvault
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &datakeys datakeys
+
+initialData:
+  - databaseName: *keyvault
+    collectionName: *datakeys
+    documents: []
```

Comment:
Is this relevant if all of the tests expect client errors?

Reply:
I suppose not. Will remove.